tests/async_engine/test_async_llm_engine.py

```diff
@@ -23,15 +24,21 @@ def __init__(self):
         self.add_request_calls = 0
         self.abort_request_calls = 0
         self.request_id = None
+        # Ugly, remove dependency when possible
+        self.parallel_config = ParallelConfig(1, 1, False)
```

Comment:
is it still needed?

Reply:
It's still needed, we use it in async engine here:

https://github.com/vllm-project/vllm/pull/4412/files#diff-4bd825485b5162cf8021da41a2ebd3d4026929e617d1137f69bbbe4bda0ee643R552-R553